concourse/tasks/generate-test-config.yml

```diff
@@ -17,11 +17,11 @@ run:
     - -e
     - -c
     - |
-      ruby -ryaml -rjson -e "
-        config = YAML.load(STDIN)['properties']['$TEST_PROPERTIES']
-        config['admin_user'] = File.read('admin-creds/username').strip()
-        config['admin_password'] = File.read('admin-creds/password').strip()
-        puts JSON.pretty_generate(config)
-      " \
-      < cf-manifest/cf-manifest.yml \
-      > test-config/config.json
+      bosh interpolate --var-errs \
+        --var-file=admin_user=admin-creds/username \
+        --var-file=admin_password=admin-creds/password \
```

Comment:
Does this result in these values actually ending up in the resulting output. The test config manifests don't seem to use these variables - they have "admin" hardcoded for the user, and use `((secrets_uaa_admin_password))` for the password.

Reply:
good point, I will have a look in some minutes

Reply:
fixed